app/uk/gov/hmrc/cgtpropertydisposals/models/des/returns/DesReturnDetails.scala

```diff
@@ -61,8 +63,11 @@ object DesReturnDetails {
     )
   }
 
-  private def getSource(submitReturnRequest: SubmitReturnRequest): String =
-    submitReturnRequest.agentReferenceNumber.fold("self digital")(_ => "agent digital")
+  private def getSource(submitReturnRequest: SubmitReturnRequest): String = {
+    val timestamp = Instant.now.getEpochSecond.toString
```

Comment:
this should be an empty string if it's not a further return

Reply:
fixed